modules/ztp-preparing-the-ztp-git-repository.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-preparing-the-ztp-git-repository_{context}"]
+= Preparing the ZTP Git repository
+
+Create a Git repository for hosting site configuration data. The zero touch provisioning (ZTP) pipeline requires read access to this repository.
+
+.Procedure
+
+. Create a directory structure with separate paths for the SiteConfig and PolicyGenTemplate custom resources.
```

Comment:
s/SiteConfig and PolicyGenTemplate custom resources/`SiteConfig` and `PolicyGenTemplate` custom resources (CR)

Reply:
changed